rfc6962-bis.xml

```diff
@@ -541,9 +541,7 @@ ranges.
         v2(1), (255)
     } Version;
 
-    struct {
-        opaque key_id[HASH_SIZE];
-    } LogID;
+    opaque LogID&lt;1..2^8-1&gt;;
```

Comment:
That <1..2^8-1> means 1 byte is used to hold the variable length number of bytes of the DER-encoded OID.  I've never seen a DER-encoded OID use up anywhere near 255 bytes!